tuner.c

```diff
@@ -486,17 +486,21 @@ bool cSatipTuner::UpdatePids(bool forceP)
      cString uri = cString::sprintf("%sstream=%d", *GetBaseUrl(*streamAddrM, streamPortM), streamIdM);
      bool useci = (SatipConfig.GetCIExtension() && currentServerM.HasCI());
      bool usedummy = currentServerM.IsQuirk(cSatipServer::eSatipQuirkPlayPids);
+     bool paramadded = false;
      if (forceP || usedummy) {
         if (pidsM.Size())
            uri = cString::sprintf("%s?pids=%s", *uri, *pidsM.ListPids());
```

Comment:
Could you add ```paramadded ? "&" : "?"``` logic also into this line? I know, it will be always ```false```, but any future modification will be adapted automatically afterwards.
Please, amend this also into the existing commit.